data/programs/accelerationProgramFaq.tsx

```diff
@@ -0,0 +1,51 @@
+import { Faq } from "@/lib/types"
+
+export const accelerationProgramFaq: Faq[] = [
+  {
+    question: "Who can apply?",
+    answer:
+      "The program is open to alumni of our entry level programs (e.g. Contributions Program, ZK Playground) and other applicants at beginner to intermediate levels with programmable cryptography.",
+  },
+  {
+    question: "What platform does the program use?",
+    answer:
+      "We will primarily utilize a Github repository for managing documents and staging of the grant proposals, grantees, and their progress. Stakeholders involved can utilize Github issues and pull requests to comment and collaborate.",
+  },
+  {
+    question: "How does the grant funding work?",
+    answer: (
+      <>
+        <span>
+          While the core funding this program comes through PSE via the Ethereum
+          Ecosystem Support Program (ESP), some bounties are supported by
+          specific teams.
+        </span>
```

Comment:
What are these spans supposed to do?
It does not look as expected here:
<img width="719" alt="image" src="https://github.com/privacy-scaling-explorations/website-v2/assets/735288/7402461d-5be1-4cd4-9ac6-81b5a26a552d">

Reply:
Thank you for your feedback!
I have fixed the spacing, and the different `span` will correctly create a gap between the paragraphs